### PR TITLE
GitHub request template

### PR DESCRIPTION
**Pull request by saryazdi**
- Added `CONTRIBUTING.md`
- Added `CODE_OF_CONDUCT.md`
- Added PR/issue templates in `.github/`
- Fixed documentation tutorials' colab notebook links

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- [ ] Documentations: did you build documentation ? `cd docs/ && sphinx-build . _build`
+- [ ] Implementation: is your code well commented and follow conventions ? `black gradslam/ && flake8 gradslam/`
+- [ ] Docstrings & Typing: has your code documentation and typing ?
+- [ ] Update notebooks & documentation if necessary
+
+### gradslam Team
+<details>
+  <summary>gradslam team workflow</summary>
+  
+  - [ ] Assign correct label
```
Comment (krrish94) [inline .github/pull_request_template.md:34]: I would rename this to `- [ ] Triage` and introduce a new label that says `triaged` (i.e., all issues that have the `triaged` label have been assigned correct labels; this is useful when searching for unlabelled issues)

**Review by krrish94 — COMMENTED** [review]
LGTM, one minor remark

File: .github/pull_request_template.md (hunk 1 above)
Comment (saryazdi) [reply .github/pull_request_template.md]: Makes sense, modified

**Review by krrish94 — APPROVED** [review]
thanks for the edit

**Pull request merged**